test/e2e/tests/risingwave/tests.sh

```diff
@@ -159,7 +159,7 @@ function test::risingwave::check_status_with_simple_queries() {
   local frontend_service_port
   frontend_service_port=$(k8s::kubectl get svc/"${E2E_RISINGWAVE_NAME}-frontend" -o jsonpath='{.spec.ports[?(@.name=="service")].port}')
 
-  testenv::util::psql -h "${E2E_RISINGWAVE_NAME}-frontend.${E2E_NAMESPACE}" -p "${frontend_service_port}" -d dev -U root <<EOF
+  testenv::util::psql -X -v ON_ERROR_STOP=1 -h "${E2E_RISINGWAVE_NAME}-frontend.${E2E_NAMESPACE}" -p "${frontend_service_port}" -d dev -U root <<EOF
```

Comment:
Really good idea with the `ON_ERROR_STOP`! Why are you passing the `-X`?

```
-X, --no-psqlrc          do not read startup file (~/.psqlrc)
```

Why is reading the conf file bad?

Reply:
It possibly changes the behaviour of `psql`. Passing `-X` makes sure that it won't read any other configurations so the command is always reproducible.

Reply:
Actually, I just followed the suggestions from https://stackoverflow.com/questions/4480381/how-can-i-stop-a-postgres-script-when-it-encounters-an-error 🤣